### ISSUE TITLE
Failed precondition: VOCdevkit

### ISSUE THREAD

**Issue by jiyongma** [DrSleep/tensorflow-deeplab-resnet#173]
2018-03-31 15:16:50.186012: W tensorflow/core/framework/op_kernel.cc:1158] Failed precondition: VOCdevkit
2018-03-31 15:16:50.186013: W tensorflow/core/framework/op_kernel.cc:1158] Failed precondition: VOCdevkit
step 1 	 loss = 0.252, (1.038 sec/step)
step 2 	 loss = 0.200, (0.209 sec/step)
2018-03-31 15:16:51.357591: W tensorflow/core/framework/op_kernel.cc:1158] Out of range: FIFOQueue '_1_create_inputs/batch/fifo_queue' is closed and has insufficient elements (requested 4, current size 0)
	 [[Node: create_inputs/batch = QueueDequeueManyV2[component_types=[DT_FLOAT, DT_UINT8], timeout_ms=-1, _device="/job:localhost/replica:0/task:0/cpu:0"](create_inputs/batch/fifo_queue, create_inputs/batch/n)]]
2018-03-31 15:16:51.357623: W tensorflow/core/framework/op_kernel.cc:1158] Out of range: FIFOQueue '_1_create_inputs/batch/fifo_queue' is closed and has insufficient elements (requested 4, current size 0)

**Comment by DrSleep** [comment]
what script are you running? did you check that all the images are present? more details will definitely help

**Comment by jiyongma** [comment]
When I use all the VOC data sets for training, I can run out perfectly and run the final segmentation results, but this error occurs when the number of data sets I train decreases.

**Comment by jiyongma** [comment]
Thank you ,please give me some suggestions.

**Comment by DrSleep** [comment]
what do you mean by this?

> when the number of data sets I train decreases

**Comment by jiyongma** [comment]
When I set up a epoch, I could run. When I set up multiple epoch, I just ran a epoch and stopped running.

self.queue = tf.train.slice_input_producer([self.images, self.labels],shuffle=False,num_epochs=2)

**Comment by jiyongma** [comment]
When the number of txt  is reduced，/deeplab/dataset/train.txt    10582  Ｉchange it  to 10580 /less

**Comment by DrSleep** [comment]
that is quite strange.

do you change anything else?
